databricks/koalas/frame.py

```diff
@@ -399,6 +400,44 @@ def dropna(self, axis=0, how='any', thresh=None, subset=None, inplace=False):
         else:
             raise NotImplementedError("dropna currently only works for axis=0 or axis='index'")
 
+    def fillna(self, value=None, axis=None, inplace=False):
+        """Fill NA/NaN values.
+
+        :param value: scalar, dict, Series
+                    Value to use to fill holes. alternately a dict/Series of values
+                    specifying which value to use for each column.
+                    DataFrame is not supported.
+        :param axis: {0 or `index`}
+                    1 and `columns` are not supported.
+        :param inplace: boolean, default False
+                    Fill in place (do not create a new object)
+        :return: :class:`DataFrame`
+        """
```

Comment:
Can we copy and paste doctests?

Reply:
I mean here https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.fillna.html#pandas.DataFrame.fillna

Reply:
Do you mean examples?

Reply:
Yea, doc link above.https://github.com/pandas-dev/pandas/blob/master/pandas/core/generic.py#L5984

Ideally if we replace pd to ks, it should work.if it doesn't work we can fix the doctest/docstrings or do it later

Reply:
Good! I skipped `See also` part.